vmdb/app/models/orchestration_stack_openstack_infra.rb

```diff
@@ -0,0 +1,13 @@
+class OrchestrationStackOpenstackInfra < OrchestrationStack
+  def raw_update_stack(options)
+    ext_management_system.with_provider_connection(:service => "CloudFormation") do |connection|
```

Comment:
@bzwei Btw. the implementation for Openstack and OpenstackInfra will be different.